appscale/tools/appscale_tools.py

```diff
@@ -480,9 +482,12 @@ def gather_logs(cls, options):
     failures = False
     for ip in all_ips:
       # Get the logs from each node, and store them in our local directory
-      local_dir = "{0}/{1}".format(options.location, ip)
+      local_dir = os.path.join(location, ip)
       os.mkdir(local_dir)
 
+      if ip == login_host:
+        os.symlink(local_dir, os.path.join(location, "login-host"))
```

Comment:
Since login-host is not a role, and we don't use anywhere else, could we change this to something a user can relate more? Is this the load-balancer perhaps?